### ISSUE TITLE
[NuGet.org Bug]: no images are shown in package readme from https://github.com/:user/:repo/assets/.../:file

### ISSUE THREAD

**Issue by TheSquidCombatant** [NuGet/NuGetGallery#9621]
### Impact

It bothers me. A fix would be nice

### Describe the bug

Hello!

I have a bug in my package readme: https://www.nuget.org/packages/TheSquid.Numerics.Extensions/.
No images from domain: https://github.com/:user/:repo/assets/.../:file.

I see two warning messages: " This documentation had some images automatically rewritten to use secure links and may be broken." and "Some images are not displayed as they are not from [trusted domains](https://aka.ms/nuget-org-readme#allowed-domains-for-images-and-badges).".

But there are just assets from my public repository in github: https://github.com/TheSquidCombatant/TheSquid.Numerics.Extensions. Looking forward to the result.

I suggest to add github addresses of this type to the list of allowed domains:
https://aka.ms/nuget-org-readme#allowed-domains-for-images-and-badges

P.S.: please notify me if I did something wrong when uploading images.

### Repro Steps

I have a bug in my package readme: https://www.nuget.org/packages/TheSquid.Numerics.Extensions/.

No images from domain: https://github.com/:user/:repo/assets/.../:file.

I see two warning messages: " This documentation had some images automatically rewritten to use secure links and may be broken." and "Some images are not displayed as they are not from [trusted domains](https://aka.ms/nuget-org-readme#allowed-domains-for-images-and-badges).".

### Expected Behavior

I suggest to add github addresses of type
    https://github.com/:user/:repo/assets/.../:file.
to the list of image allowed domains:
    https://aka.ms/nuget-org-readme#allowed-domains-for-images-and-badges

**Comment by vipwan** [comment]
image path will redirect to https://user-images.githubusercontent.com/........

**Comment by TheSquidCombatant** [comment]
> image path will redirect to https://user-images.githubusercontent.com/........

Yep. But when I drag and drop image file into my readme.md through web editor, image uploads by default with path https://github.com/{my_profile}/{my_repository}/assets/...

How could I upload images into https://user-images.githubusercontent.com/... ???

**Comment by sailro** [comment]
@TheSquidCombatant as a workaround, use https://wheregoes.com/ to see all redirects for your GitHub image. (Juste pas your url). This will display the original "githubusercontent" url.
